sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParserUtils.scala

```diff
@@ -259,4 +260,48 @@ object ParserUtils {
       }
     }
   }
+
+  /**
+   * Converts the given parse tree to an alias by
```

Comment:
`Normalizes the expression parser tree to a SQL string which will be used to generate expression alias`